scripts/unit-tests.sh

```diff
@@ -19,6 +27,3 @@ fi
 # Run unit tests with coverage
 uv run --python "$PYTHON_VERSION" --with-editable . --group unit \
     coverage run --source=llama_stack -m pytest -s -v tests/unit/ "$@"
-
-# Generate HTML coverage report
-uv run --python "$PYTHON_VERSION" coverage html -d htmlcov-$PYTHON_VERSION
```

Comment:
Curious - why do we run unit tests with coverage above and also run coverage in the `cleanup` function?

Reply:
@nathan-weinberg good question. The command `coverage run` with `pytest` will run all the tests and collect the coverage data in a `.coverage` file. Then, when the script exits, the `coverage html` command is ran to generate the report.

Note, when running `coverage html` this doesn't run the tests again, it just uses the already collected data and generates the report.